update/src/Pos/Update/Poll/Types.hs

```diff
@@ -119,6 +130,17 @@ type LocalVotes = HashMap UpId (HashMap PublicKey UpdateVote)
 -- Proposal State
 ----------------------------------------------------------------------------
 
+-- | Extra data required by wallet, stored in UndecidedProposalState
+data UpsExtra = UpsExtra
+    { ueProposedBlk :: !HeaderHash
+    -- ^ Block in which this update was proposed
+    } deriving (Show, Generic, Eq)
+
+deriveSimpleBi ''UpsExtra [
+  Cons 'UpsExtra [
```

Comment:
We should try to stick to the 4 space indent standard.

Reply:
Not sure what happened here. Will fix these.